src/internal/connector/graph/errors.go

```diff
@@ -313,3 +314,32 @@ func appendIf(a []any, k string, v *string) []any {
 
 	return append(a, k, *v)
 }
+
+// MalwareInfo gathers potentially useful information about a malware infected
+// drive item, and aggregates that data into a map.
+func MalwareInfo(item models.DriveItemable) map[string]any {
```

Comment:
Minor - use const for `created_by` and other fields. Add a simple unit test to validate this method

Reply:
done